overrides.rb

```diff
@@ -85,9 +85,13 @@
     if pkg.kind_of?(Autobuild::CMake)
         pkg.post_import do
             Rock.update_cmake_build_type_from_tags(pkg)
+            if File.directory?(File.join(pkg.srcdir, 'viz'))
+                pkg.env_add_path 'VIZKIT_PLUGIN_RUBY_PATH', File.join(pkg.prefix, 'lib')
+            end
         end
         pkg.define "ROCK_TEST_ENABLED", pkg.test_utility.enabled?
         pkg.define "CMAKE_EXPORT_COMPILE_COMMANDS", "ON"
+        pkg.env_add_path 'QT_PLUGIN_PATH', File.join(pkg.prefix, 'lib', 'qt')
```

Comment:
The prefix paths will only exist after install, for which there is no hook in autobuild/autoproj. Autoproj's environment object will filter out the paths that won't exist on export.

Note that for VIZKIT_PLUGIN_RUBY_PATH, we only check for the `viz` directory in the source, not the prefix directory.